### PR TITLE
ec2_eip_facts - Add link to describe-address filters doc

### PR DESCRIPTION
**Pull request by dthvt**
##### SUMMARY

Adds link to AWS API filter documentation for ec2_eip_facts,

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

ec2_eip_facts

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by gundalow** [comment]
bot_status

File: lib/ansible/modules/cloud/amazon/ec2_eip_facts.py [hunk 1]
```diff
@@ -18,8 +18,9 @@
 options:
   filters:
     description:
-      - A set of filters to use. Each filter is a name:value pair. The value
-        may be a list or a single element.
+      - A dict of filters to apply. Each dict item consists of a filter key and filter
+        value.  See https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-addresses.html#options
```
Comment (gundalow) [inline lib/ansible/modules/cloud/amazon/ec2_eip_facts.py]: Adding `U(...)` will make this a URL
```suggestion
        value.  See U(https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-addresses.html#options)
```

**Comment by dthvt** [comment]
@gundalow thanks - the module I copied from didn't have the U() either so I'll keep an eye out for those.  (Forget which one I used for reference now, unfortunately.)

**Comment by gundalow** [comment]
Thanks. We are continually making improvements to how modules are documented, often small though they add up over time.

**Review by gundalow — APPROVED** [review]

**Pull request merged**